tensorly/decomposition/tests/test_cp.py

```diff
@@ -13,6 +14,29 @@
 from ...testing import assert_array_equal, assert_
 
 
+def tucker_congruence(A, B):
```

Comment:
Should we just call it congruence_coefficient and add a short docstring ? 
I find it very useful, do you think it may be worth making it available more generally?
Or should it be used just under the hood, e.g to "align" CP decompositions?